data/campaigns/fri04.wmf/scripting/mission_thread.lua

```diff
@@ -87,6 +107,11 @@ function mission_thread()
    end
    sleep(2000)
    campaign_message_box(intro_3)
+   run(hurry_up)
+   if campaign_data.payment then run(function()
+      sleep(1000 * 60 * 10)
+      campaign_message_box(legate_expands)
```

Comment:
Suggestion make this dependent on a specific number of military buildings of tzhe empire rather then just a timer and let it run for some while